gradle/libs.versions.toml

```diff
@@ -1,8 +1,8 @@
 [versions]
 androidGradlePlugin = '7.2.2'
-kotlin = '1.7.21'
-kotlinCoroutines = '1.6.4'
-ktor = '2.0.0'
+kotlin = '1.8.21'
+kotlinCoroutines = '1.7.1'
```

Comment:
A newer version of coroutines is available: 1.7.3
https://github.com/Kotlin/kotlinx.coroutines/releases